Tests/SwerverTests/RequestSpec/HttpRequestParserSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class HttpRequestParserSpec: QuickSpec {
+    override func spec() {
+        describe("An HTTP Request Parser") {
+            var httpRequestParser: HttpRequestParser!
+            
+            beforeEach {
```

Comment:
This `beforeEach` does not seem needed, especially given there is only one test.